src/components/Modal.js

```diff
@@ -0,0 +1,55 @@
+import { Component } from 'react'
+import { div, hh } from 'react-hyperscript-helpers'
+import * as Style from 'src/libs/style'
+
+
+/**
+ * @param onDismiss
+ * @param title
+ * @param children
+ * @param showCancel=true
+ * @param okButton
+ */
+export default hh(class Modal extends Component {
```

Comment:
Doesn't appear to be used, so hard to test for code review. In the future, I'm totally fine with having a button who's sole purpose is to try out a control like this. We could enable it via feature flag if we want prod to be clean.